nightwatch.conf.js

```diff
@@ -79,7 +79,7 @@ module.exports = {
           resolution: '1366x768',
           buildName: `QA2_CHROME ${dayjs().format('YYYY-MM-DD')} JIRA: ${process.env.TEST_EXECUTION_KEY}`,
           local: 'false',
-          idleTimeout: 300,
+          idleTimeout: 1200,
```

Comment:
**question**: Do we need this also for other browsers? And why we need that?

Reply:
reverted to 300. was initially changed to address timeout issue, but source of error was from a circleci setting. see commit 86fd86556e4dd20494b0efe531b5a32315764561